samples/otlptraceexport/src/app-grpc-export.ts

```diff
@@ -33,12 +33,12 @@ async function getAuthenticatedClient(): Promise<AuthClient> {
 // Express App that exports traces via gRPC with protobuf
 async function main() {
   const authenticatedClient: AuthClient = await getAuthenticatedClient();
+  const sslCreds: ChannelCredentials = ChannelCredentials.createSsl();
 
   const sdk = new NodeSDK({
     traceExporter: new OTLPTraceExporter({
-      credentials: credentials.combineChannelCredentials(
-        credentials.createSsl(),
-        credentials.createFromGoogleCredential(authenticatedClient)
```

Comment:
IMO this is more readable and less complex, so would be my preference if they are functionally equivalent.

Reply:
I can close this PR as I do not have any particular preference in this case.